ee/tasks/webhooks_ee.py

```
@@ -12,41 +12,53 @@
-            continue
-        message_text, message_markdown = get_formatted_message(action, _event, site_url,)
-        if determine_webhook_type(team) == "slack":
-            message = {
-                "text": message_text,
-                "blocks": [{"type": "section", "text": {"type": "mrkdwn", "text": message_markdown},},],
-            }
-        else:
-            message = {
-                "text": message_markdown,
-            }
-        requests.post(team.slack_incoming_webhook, verify=False, json=message)
+        actionFilters = {"team_id": team_id}
+        if not is_zapier_available:
+            if not team.slack_incoming_webhook:
+                return  # Exit this task if neither Zapier nor webhook URL are available
+            else:
+                actionFilters["post_to_slack"] = True  # We only need to fire for actions that are posted to webhook URL
+        actions = cast(Sequence[Action], Action.objects.filter(**actionFilters).all())
 
-    _event.delete()
+        for action in actions:
+            qs = Event.objects.filter(pk=postgres_event.pk).query_db_by_action(action)
+            if not qs:
+                continue
+            # REST hooks
+            if is_zapier_available:
+                action.on_perform(postgres_event)
+            # webhooks
+            if team.slack_incoming_webhook:
+                message_text, message_markdown = get_formatted_message(action, postgres_event, site_url,)
+                if determine_webhook_type(team) == "slack":
+                    message = {
+                        "text": message_text,
+                        "blocks": [{"type": "section", "text": {"type": "mrkdwn", "text": message_markdown},},],
+                    }
+                else:
+                    message = {
+                        "text": message_markdown,
+                    }
+                requests.post(team.slack_incoming_webhook, verify=False, json=message)
+    except:
```

We swallow the error here now. Should we notify sentry?